Assets/Scripts/UI/InGame/Graphics/GeneralGraphics.cs

```diff
@@ -0,0 +1,287 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Rendering;
+using Newtonsoft.Json;
+namespace Assets.Scripts.UI.InGame
+{
+	public class GeneralGraphics : MonoBehaviour {
+
+		[SerializeField] public Dropdown textureQuality;
+		[SerializeField] public Dropdown shadowRes;
+		[SerializeField] public Dropdown antiAliasing;
+		[SerializeField] public Dropdown shadows;
+		[SerializeField] public Toggle vSync;
+		[SerializeField] public Toggle softParticles;
+
+		[SerializeField] public Toggle customSettings;
+
+		public Dropdown TextureQuality
+		{ 
+			get { return textureQuality; }
+			set { textureQuality = value; }
+		}
+		public Dropdown ShadowRes
+		{ 
+			get { return shadowRes; }
+			set { shadowRes = value; }
+		}
+		public Dropdown AntiAliasing
+		{
+			get { return antiAliasing; }
+			set { antiAliasing = value; }
+		}
+		public Dropdown Shadows
+		{
+			get { return shadows; }
+			set { shadows = value; }
+		}
+		public Toggle VSync
+		{
+			get { return vSync; }
+			set { vSync = value; }
+		}
+		public Toggle SoftParticles
+		{
+			get { return softParticles; }
+			set { softParticles = value; }
+		}
+
+		public Toggle CustomSettings
+		{
+			get { return customSettings; }
+			set { customSettings = value; }
+		}
+
+		private void Update() {
+			
+			if(!CustomSettings.isOn)
+			{
+				// shadow res
+				if(QualitySettings.shadowResolution == ShadowResolution.Low)
```

Comment:
Enums are "Named Integers". ShadowResolution.Low has the value of 0, so you can just do:
`ShadowRes.value = (int) QualitySettings.shadowResolution`

Apply the same to the other enums you are using. Will save you about 50 lines of code.